sa-test.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
```

Comment:
There are the old test files when JavaScript was used, please let's use only Python.

Reply:
Ok, deleted.